### DISCUSSION TITLE
Windows on ARM

### DISCUSSION THREAD

**Discussion by Impatient0717** [HotCakeX/Harden-Windows-Security#257] [General]
Firstly, thank you for your continued efforts with this hardening guide - has been incredibly useful. 
    With all the hype concerning Qualcomm making the CPUs for the next iteration of windows laptops, what effect - if any - do you see this change in CPU architecture having on security?

**Comment by HotCakeX** [comment]
Hi,
my pleasure :)

The most important effect is having access to the Microsoft's Pluton chip. It's something only AMD and ARM CPUs have incorporated so far and Intel CPUs don't have it yet.

[Pluton](https://www.microsoft.com/en-us/security/blog/2020/11/17/meet-the-microsoft-pluton-processor-the-security-chip-designed-for-the-future-of-windows-pcs/) is important because it [addresses security needs](https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/defending-against-ransomware-with-microsoft-defender-for/ba-p/3243941) like booting an operating system securely even against firmware threats and storing sensitive data safely even against physical attacks.

It's also [available](https://learn.microsoft.com/en-us/windows/security/hardware-security/pluton/microsoft-pluton-security-processor) on Xbox Series X/S to provide the security they need against the most severe attacks threatening that [platform](https://www.youtube.com/watch?v=quLa6kzzra0).

**Reply by Impatient0717** [reply]
Hi,
   Thank you for the response. The linked articles are interesting reads. All the best.
